### PR TITLE
[NOT AZURE] Azure Stack Network Admin SDK

### PR DESCRIPTION
**Pull request by deathly809**
## Description

This is for Azure Stack **ONLY**.  There seems to be an issue when generating the SDK remotely for the repo some reason.  I have message the Autorest team and ask them for help but no response.

https://github.com/Azure/azure-rest-api-specs/pull/2813

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
@deathly809 Please fix compilation errors here

File: src/SDKs/_metadata/azsadmin_resource-manager_network.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/network/readme.md --csharp --csharp-sdks-folder=E:\github\azure-sdk-for-net\src\AzureStack\NetworkAdmin\Network.Admin --version=latest --reflect-api-versions
+
+AutoRest code generation for https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/network/readme.md failed. Please try again
```
Comment (dsgouda) [inline src/SDKs/_metadata/azsadmin_resource-manager_network.txt]: Please generate the code using generate.ps1 or generate.cmd and commit those changes.
Looks like autorest failed to generate the code from config file here

File: src/AzureStack/NetworkAdmin/Network.Admin/Microsoft.AzureStack.Management.Network.Admin.csproj [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (shahabhijeet) [inline src/AzureStack/NetworkAdmin/Network.Admin/Microsoft.AzureStack.Management.Network.Admin.csproj]: @deathly809 missing comment around the includes.
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L2
Also missing the include that is required in the footer of the file, model your csproj using this
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L22

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/AzureStack/NetworkAdmin/Network.Admin.Tests/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+[assembly: AssemblyProduct("Azure Stack Network Admin Tests")]
+[assembly: AssemblyCopyright("Copyright © Microsoft 2017")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+[assembly: ComVisible(false)]
+
+[assembly: Guid("1b24bc4a-6e4d-4feb-b5d5-c9684c60a308")]
+
+// [assembly: AssemblyVersion("1.0.*")]
```
Comment (shahabhijeet) [inline src/AzureStack/NetworkAdmin/Network.Admin.Tests/Properties/AssemblyInfo.cs]: remove commented code

File: src/AzureStack/NetworkAdmin/Network.Admin/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("Microsoft Azure .NET SDK")]
+[assembly: AssemblyCopyright("Copyright © Microsoft")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+[assembly: NeutralResourcesLanguage("en")]
+
+// Version information for an assembly consists of the following four values:
```
Comment (shahabhijeet) [inline src/AzureStack/NetworkAdmin/Network.Admin/Properties/AssemblyInfo.cs]: remove commented code

File: src/AzureStack/NetworkAdmin/Network.Admin/Microsoft.AzureStack.Management.Network.Admin.csproj [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    <Description>Provides developers with libraries for the network admin under AzureStack to manage network.</Description>
+    <VersionPrefix>0.1.0-preview</VersionPrefix>
+    <AssemblyName>Microsoft.AzureStack.Management.Network.Admin</AssemblyName>
+    <PackageTags>Microsoft Azure Stack;Network;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
+    <PackageReleaseNotes>This is a preview release of the AzureStack Network Admin SDK. Included with this release is support for gathering information about your AzureStack infrastructure and performing administrative operations.</PackageReleaseNotes>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+</Project>
```
Comment (shahabhijeet) [inline src/AzureStack/NetworkAdmin/Network.Admin/Microsoft.AzureStack.Management.Network.Admin.csproj]: missing footer includes

File: src/AzureStack/NetworkAdmin/Network.Admin/Microsoft.AzureStack.Management.Network.Admin.csproj (hunk 2 above)
Comment (deathly809) [reply src/AzureStack/NetworkAdmin/Network.Admin/Microsoft.AzureStack.Management.Network.Admin.csproj]: Fixed.

File: src/AzureStack/NetworkAdmin/Network.Admin.Tests/Properties/AssemblyInfo.cs (hunk 3 above)
Comment (deathly809) [reply src/AzureStack/NetworkAdmin/Network.Admin.Tests/Properties/AssemblyInfo.cs]: Fixed.

File: src/AzureStack/NetworkAdmin/Network.Admin/Properties/AssemblyInfo.cs (hunk 4 above)
Comment (deathly809) [reply src/AzureStack/NetworkAdmin/Network.Admin/Properties/AssemblyInfo.cs]: Fixed.

File: src/AzureStack/NetworkAdmin/Network.Admin/Microsoft.AzureStack.Management.Network.Admin.csproj (hunk 5 above)
Comment (deathly809) [reply src/AzureStack/NetworkAdmin/Network.Admin/Microsoft.AzureStack.Management.Network.Admin.csproj]: Fixed.

**Comment by dsgouda** [comment]
@deathly809 yes It could have been a machine issue, restarted the build, please wait.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**